File: files/CH-079.R
library(tidyverse)
library(readxl)

path = "files/CH-079 Remove Blank Columns.xlsx"
input = read_excel(path, range = "B2:I6")
test  = read_excel(path, range = "K2:N6")

result = input %>%
  select(-where(~all(is.na(.))))

identical(result, test)
# [1] TRUE

File: files/000-099/079/CH-079.py
import pandas as pd

path = "CH-079 Remove Blank Columns.xlsx"

input = pd.read_excel(path, usecols="B:I", skiprows=1)
test  = pd.read_excel(path, usecols="K:N", skiprows=1)
test.columns = test.columns.str.replace('.1', '')

result = input.loc[:, input.columns[~input.isnull().all()]]
print(result.equals(test)) # True